main.tf

```diff
@@ -87,8 +87,23 @@ resource "aws_lambda_function" "logs_to_datadog" {
       DD_STORE_FAILED_EVENTS = var.store_failed_events
       DD_S3_BUCKET_NAME      = module.datadog_serverless_s3.bucket_name
       ## Filter out lambda platform logs
-      DD_LOGS_CONFIG_PROCESSING_RULES = "[{\"type\": \"exclude_at_match\", \"name\": \"exclude_start_and_end_logs\", \"pattern\": \"(START|END) RequestId\"}]"
-      EXCLUDE_AT_MATCH                = "\"(START|END) RequestId:\\s"
```

Comment:
This never worked probaly because of the extra `"` , but fwiw `EXCLUDE_AT_MATCH` is really for 1 regex pattern, and `DD_LOGS_CONFIG_PROCESSING_RULES` allows for multiple rules